application.go

```diff
@@ -480,7 +478,6 @@ func (r *marathonClient) DeleteApplication(name string) (*DeploymentID, error) {
 // RestartApplication performs a rolling restart of marathon application
 // 		name: 		the id used to identify the application
 func (r *marathonClient) RestartApplication(name string, force bool) (*DeploymentID, error) {
-	glog.V(debugLevel).Infof("restarting the application: %s, force: %s", name, force)
```

Comment:
Why not convert this and the following removals in this file into `log.*` statements?